test/e2e/cypress/e2e/sap_systems_overview.cy.js

```diff
@@ -344,4 +344,54 @@ context('SAP Systems Overview', () => {
       cy.contains(sapSystemNwd.sid).should('exist');
     });
   });
-});
+
+  describe("Move application instance", () => {
+    const nwdSystem = {
+      sid: 'NWD',
+      id: 'f534a4ad-cef7-5234-b196-e67082ffb50c',
+      hostId: '9a3ec76a-dd4f-5013-9cf0-5eb4cf89898f',
+      instanceNumber: '02',
+    }
+
+    before(() => {
+      cy.contains(nwdSystem.sid).should('exist');
+
+      cy.get('table.table-fixed > tbody > tr').eq(0).click();
+    });
+
+    after(() => {
+      cy.loadScenario('sap-systems-overview-revert-not-moved')
+      cy.get('table.table-fixed ')
+        .contains('Clean up', { timeout: 15000 });
+      cy.deregisterInstance(nwdSystem.id, nwdSystem.hostId, nwdSystem.instanceNumber)
+    })
+
+    it("should move a clustered application instance", () => {
+      cy.loadScenario('sap-systems-overview-moved');
+
+      cy.get('table.table-fixed > tbody > tr')
+        .eq(1)
+        .find('div.table-row-group')
+        .eq(0)
+        .find('div.table-row')
+        .its('length')
+        .should('eq', 4);
+
+        cy.contains('vmnwdev01').should('not.exist');
```

Comment:
I would put this `vmnwdev01` in the `nwdSystem` const.

Reply:
done